doc/releases/reef.rst

```diff
@@ -4,6 +4,24 @@ Reef
 
 Reef is the 18th stable release of Ceph. It is named after the reef squid (Sepioteuthis).
 
+v18.2.6 Reef
+============
+
+This is the sixth hotfix release in the Reef series. We recommend that all users update to this release.
```

Comment:
do we want to just say `hotfix release`? The `sixth` makes it sound to me like we've done 6 hotfix releases rather than just 6 reef point releases

Reply:
see if you like this better